doc/source/development/policies.rst

```diff
@@ -0,0 +1,51 @@
+.. _develop.policies:
+
+********
+Policies
+********
+
+.. _policies.version:
+
+Version Policy
+~~~~~~~~~~~~~~
+
+.. versionchanged:: 1.0.0
+
+Pandas uses a variant of `SemVer`_ to govern deprecations, API compatibility, and version numbering.
```

Comment:
maybe in a future iteration, we could be explicit on the differences to semver and explain the reasoning for the differences in that context.